### PR TITLE
Make it easy to update docs with the single command

### PR DESCRIPTION
**Pull request by devoto13**
See separate commits for details.

### CODE REVIEW

File: sync-docs.sh [hunk 1]
```diff
@@ -12,16 +13,23 @@ if [ -z "$VERSION" ]; then
...
-#cd $KARMA_REPO
-#git fetch $REMOTE
-#git checkout $REMOTE/$BRANCH
+# get latest Karma repo and switch to requested branch or tag
+cd $KARMA_REPO
+if [ -d "$KARMA_REPO" ]; then
+  echo "Fetching Karma repository updates..."
+  git fetch
+else
+  echo "Clonning Karma repository..."
```
Comment (johnjbarton) [inline sync-docs.sh:25]: Cloning

File: sync-docs.sh [hunk 2]
```diff
@@ -12,16 +13,23 @@ if [ -z "$VERSION" ]; then
...
+  git clone https://github.com/karma-runner/karma.git
+fi
+echo "Switching Karma repository to $BRANCH"
```
Comment (johnjbarton) [inline sync-docs.sh:28]: maybe Karma repository -> karma-runner/karma repository

File: sync-docs.sh (hunk 1 above)
Comment (devoto13) [reply sync-docs.sh]: Fixed

File: sync-docs.sh (hunk 2 above)
Comment (devoto13) [reply sync-docs.sh]: Yes, this should be more clear

**Pull request merged**